build.gradle

```diff
@@ -45,8 +45,11 @@ dependencies {
     compile 'com.google.cloud:google-cloud-storage:1.23.+'
     compile 'com.google.cloud:google-cloud-datastore:1.23.+'
     compile 'com.google.cloud:google-cloud-nio:0.42.+'
+    compile group: 'ch.qos.logback', name: 'logback-classic', version: '0.9.+'

```

Comment:
Apparently 0.9.+ is too rusty: https://mvnrepository.com/artifact/ch.qos.logback/logback-classic

Reply:
done